README.md

```diff
@@ -36,40 +36,40 @@ Its original name was Palo, developed in Baidu. After donated to Apache Software
 
 [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
 
+> **Note**   
+> Some licenses of the third-party dependencies are not compatible with Apache 2.0 License. So you need to disable
+some Doris features to be complied with Apache 2.0 License. For details, refer to the `thirdparty/LICENSE.txt`
+
 ## 2. Technology
 Doris mainly integrates the technology of Google Mesa and Apache Impala, and it is based on a column-oriented storage engine and can communicate by MySQL client.
 
-## 3. User cases
-Doris not only provides high concurrent low latency point query performance, but also provides high throughput queries of ad-hoc analysis.
+## 3. Use cases
+- Doris provides high concurrent low latency point query performance, as well as high throughput queries of ad-hoc analysis.
```

Comment:
I think it’s better to start this paragraph with this sentence “Features of Doris:”
or we can use a better subtitle to replace "Use cases"

Reply:
If you don't mind, I would like to include this part as "Highlights"(heading2) and put it in the "Apache Doris" section.

When the use case of Doris is ready, we can add this section later.

Reply:
I've applied the changes, and you can have a look at it.